src/routes/preHandlers/customers.js

```diff
@@ -79,9 +79,12 @@ exports.authorizeCustomerUpdate = async (req) => {
 exports.authorizeSubscriptionCreation = async (req) => {
   const companyId = get(req, 'auth.credentials.company._id', null);
 
-  const service = await Service.findOne({ _id: req.payload.service, company: companyId }).lean();
+  const service = await Service.countDocuments({
+    _id: req.payload.service,
+    company: companyId,
+    $or: [{ isArchived: { $exists: false } }, { isArchived: false }],
```

Comment:
On pourrait faire :
`{ isArchived: { $in: [null, false] } }` mais on perd en lisibilité, qu'en penses-tu ?

Reply:
je trouve ca moins lisible j'avoue